config/projects.yaml

```diff
@@ -398,6 +398,7 @@ twig-extensions:
                 symfony/symfony: ['4.4.*', '5.3.*', '5.4.*', '6.0.*']
 
 user-bundle:
+    doctrine_test: true
```

Comment:
Just an idea. What about defining a list of custom phpunit extensions instead of a boolean value.

Reply:
This might be a better solution indeed to avoid too many config key.

Reply:
I will try to take a look at it.

Reply:
Done.